frontend/src/main.css

```diff
@@ -35,18 +31,6 @@ a {
   width: 100%;
 }
 
-.footer-up {
-  top: 97%;
-}
-
-button {
-  white-space: nowrap;
-  display: inline-flex;
-  width: fit-content;
-  align-items: center;
-  gap: 0.5rem;
-}
```

Comment:
are you sure removing this piece of code does not have any influence anywhere in the display of all buttons in the app ?